### PR TITLE
Fix benchmarking tests

### PR DESCRIPTION
**Pull request by pskiran1**
#### What does the PR do?
- Fix `L0_perf_tensorrt_llm`
- Fix `L0_perf_vllm`

#### Checklist
- [X] PR title reflects the change and is of format `<commit_type>: <Title>`
- [ ] Changes are described in the pull request.
- [ ] Related issues are referenced.
- [ ] Populated [github labels](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels) field
- [ ] Added [test plan](#test-plan) and verified test passes.
- [x] Verified that the PR passes existing CI.
- [x] Verified copyright is correct on all changed files.
- [ ] Added _succinct_ git squash message before merging [ref](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
- [ ] All template sections are filled out.
- [ ] Optional: Additional screenshots for behavior/output changes with before/after.

#### Commit Type:
Check the [conventional commit type](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type)
box here and add the label to the github PR.
- [ ] build
- [x] ci
- [ ] docs
- [ ] feat
- [ ] fix
- [ ] perf
- [ ] refactor
- [ ] revert
- [ ] style
- [ ] test

#### Test plan:

- CI Pipeline ID: 16792324

#### Related Issues: (use one of the action keywords Closes / Fixes / Resolves / Relates to)
- closes GitHub issue: #xxx

### CODE REVIEW

**Comment by nv-kmcgill53** [comment]
It's out of scope for this PR, but it looks like there are some functions like `kill_server` which could be pulled up into the `qa/common/util.sh` script for common use.

File: qa/L0_perf_tensorrt_llm/test.sh [hunk 1]
```diff
@@ -47,40 +47,27 @@ SERVER_TIMEOUT=${SERVER_TIMEOUT:=120}
...
-        rm -rf /opt/hpcx/ompi/ /usr/local/mpi/ || {
-            echo "Failed to remove Open MPI ${CURRENT_VERSION} installation directories"
+        rm -r /opt/hpcx/ompi/ /usr/local/mpi && rm -rf /usr/lib/$(gcc -print-multiarch)/openmpi || {
+            echo "Failed to uninstall the existing Open MPI version $CURRENT_VERSION."
             exit 1
         }
-        cd ../ && rm -r openmpi-${CURRENT_VERSION}
     else
-        echo "Installed Open MPI version is not less than 5.0.1. Skipping the upgrade."
+        echo "The installed Open MPI version ($CURRENT_VERSION) is 5.0.1 or higher. Skipping the upgrade."
```
Comment (mc-nv) [inline qa/L0_perf_tensorrt_llm/test.sh:65]: DLFW uses specific version of Open MPI, by mocking and upgrading version within test we cause precedent that we start testing something that is won't be shipped to the customers.

File: qa/L0_perf_tensorrt_llm/test.sh [hunk 2]
```diff
@@ -47,40 +47,27 @@ SERVER_TIMEOUT=${SERVER_TIMEOUT:=120}
...
     cd $TENSORRTLLM_BACKEND_DIR && git lfs install && git submodule update --init --recursive
 }
 
 # Update Open MPI to a version compatible with SLURM.
 function upgrade_openmpi {
-    cd /tmp/
     local CURRENT_VERSION=$(mpirun --version 2>&1 | awk '/Open MPI/ {gsub(/rc[0-9]+/, "", $NF); print $NF}')
 
     if [ -n "$CURRENT_VERSION" ] && dpkg --compare-versions "$CURRENT_VERSION" lt "5.0.1"; then
         # Uninstall the current version of Open MPI
```
Comment (krishung5) [inline qa/L0_perf_tensorrt_llm/test.sh:59]: I understand that this is due to the SLURM MPI limitation, but agree with Misha's comment that we should be using the exact same version of the one shipped in the container. Can be future enhancement.

**Review by krishung5 — APPROVED** [review]
LGTM, great work!

**Pull request merged**